### DISCUSSION TITLE
Why would someone use MinIO instead of Ceph?

### DISCUSSION THREAD

**Discussion by elgalu** [minio/minio#11390] [Q&A] [answered]
Ceph not only provides an S3 protocol compatible object store but can also be mounted with POSIX compatible semantics without having to go through something like s3fs-fuse as it's the case with MinIO.

Sorry I don't mean to be critical but I couldn't find a proper answer, I think it's worth explaining in a place like an issue or in the docs.

**Comment by klauspost** [comment, marked as answer]
Without the marketing speech MinIO attempts to offer the best S3 compatible storage and we don't attempt to be a one-size-fit-all. Therefore filesystem access is not a priority and not encouraged.

You are welcome to contact our team at `hello@minio.io` if you would like to discuss your use-case further. We will happily enter a dialogue on whether MinIO is the right choice for you.
